driver-core/src/main/com/mongodb/internal/connection/PowerOfTwoBufferPool.java

```diff
@@ -51,27 +110,37 @@ public PowerOfTwoBufferPool() {
      * @param highestPowerOfTwo the highest power of two buffer size that will be pooled
      */
     public PowerOfTwoBufferPool(final int highestPowerOfTwo) {
+        this(highestPowerOfTwo, DEFAULT_MAX_IDLE_TIME_MINUTES, TimeUnit.MINUTES);
```

Comment:
Given that

- this is the only place where `DEFAULT_MAX_IDLE_TIME_MINUTES` is used;
- the meaning of a literal here would be clear because it is used as the value of the `maxIdleTime` parameter

introducing a constant instead of using a literal 1 does not seem helpful here.

Reply:
Done